### ISSUE TITLE
Questions about xlsx text data

### ISSUE THREAD

**Issue by ZengZ-233** [chunmeifeng/TextDiff#5]
I'm wondering if the description of the xlsx you used in your paper is a randomly set description? The following is the description I set for “description=a” and 5 randomly selected images, but the scores are significant, does it mean that the text description is actually not very influential?
![image](https://github.com/user-attachments/assets/0b2f0e9d-0e94-43c9-9890-ea8a3bc13d84)
![image](https://github.com/user-attachments/assets/d5917b88-9b73-4bee-8075-6403020f1cca)

**Comment by ZengZ-233** [comment]
oi, i get the same score with no Text, i find it is my fault that i don't use the test result but split the train set into val as the test , when i use the given test dataset officially, i get the approximate score with your paper,

**Comment by chunmeifeng** [comment]
hhh happy to see you solved your question!

ZengZ-233 ***@***.***> 于2024年11月22日周五 23:53写道：

**Comment by zmwyl** [comment]
> oi, i get the same score with no Text, i find it is my fault that i don't use the test result but split the train set into val as the test , when i use the given test dataset officially, i get the approximate score with your paper,

Hello! It seems that the author's code only contains the training process. How do you test it? Or is the training process and testing process combined in the author's code? Could you kindly guide me on this? Thank you very much！！！

**Comment by ZengZ-233** [comment]
> > oi, i get the same score with no Text, i find it is my fault that i don't use the test result but split the train set into val as the test , when i use the given test dataset officially, i get the approximate score with your paper,
> 
> Hello! It seems that the author's code only contains the training process. How do you test it? Or is the training process and testing process combined in the author's code? Could you kindly guide me on this? Thank you very much！！！

I've only used the code provided by the author, which includes the test phase, so maybe try changing the validation set to a test set?

**Comment by ZengZ-233** [comment]
> > oi, i get the same score with no Text, i find it is my fault that i don't use the test result but split the train set into val as the test , when i use the given test dataset officially, i get the approximate score with your paper,
> 
> Hello! It seems that the author's code only contains the training process. How do you test it? Or is the training process and testing process combined in the author's code? Could you kindly guide me on this? Thank you very much！！！

It's been too long since I've looked at it, I just ran it before, I'll look into it again on vacation

**Comment by zmwyl** [comment]
> > > oi, i get the same score with no Text, i find it is my fault that i don't use the test result but split the train set into val as the test , when i use the given test dataset officially, i get the approximate score with your paper,
> > 
> > 
> > Hello! It seems that the author's code only contains the training process. How do you test it? Or is the training process and testing process combined in the author's code? Could you kindly guide me on this? Thank you very much！！！
> 
> It's been too long since I've looked at it, I just ran it before, I'll look into it again on vacation

Thank you very much for your reply! I will look into it further. Thanks!
